pkg/invariants/imageregistry/disruptionimageregistry/invariant.go

```diff
@@ -47,10 +49,22 @@ func NewRecordAvailabilityOnly() invariants.InvariantTest {
 func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	var err error
 
+	namespace := "openshift-image-registry"
+
 	w.kubeClient, err = kubernetes.NewForConfig(adminRESTConfig)
 	if err != nil {
 		return err
 	}
+
+	namespaceExists, err := exutil.CheckNamespaceExists(w.kubeClient, namespace)
+	if err != nil {
+		return fmt.Errorf("unable to check namespace existence: %v", err)
+	}
+	if !namespaceExists {
+		w.notSupportedReason = "namespace openshift-image-registry not present"
+		return nil
+	}
+
```

Comment:
```suggestion
	_, err := kubeClient.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
	if apierrors.IsNotFound(err){
	    w.notSupportedReason = "namespace openshift-image-registry not present"
	    return nil
	}
	if err != nil{
	    return err
	}
```

inline is the same length